src/styles/utils/variables.scss

```diff
@@ -0,0 +1,18 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
```suggestion
```
in `*` can be only `box-sizing`, fonts styles should be in `html` selector, margin and padding should be in `body` and other tags like `p` that you use in HTML file